### PR TITLE
Added iframe clarification

### LINKED ISSUE #146: add clarifications to subtree-visibility spec about iframes
1. All elements inside iframes under a skipped `<iframe>` element are skipped.
2. The "update the rendering" steps for an iframe under a skipped `<iframe>` element is not run.

### PR DESCRIPTION
**Pull request by vmpstr**
Closes #146

### CODE REVIEW

**Comment by vmpstr** [comment]
PTAL

File: index.bs [hunk 1]
```diff
@@ -284,6 +284,22 @@ Restrictions and Clarifications {#restrictions}
       function and user gestures.
     </note>
 
+8. If an <a
+    href="https://html.spec.whatwg.org/multipage/iframe-embed-object.html#the-iframe-element">iframe</a>
+    element is [=skipped=], then the content of the frame does not paint or
```
Comment (chrishtr) [inline index.bs]:
```suggestion
    element is [=skipped=], then the content of the document inside the iframe does not paint or
```

File: index.bs [hunk 2]
```diff
@@ -284,6 +284,22 @@ Restrictions and Clarifications {#restrictions}
...
+    participate in hit testing. The user-agent should skip <a
+    href="https://html.spec.whatwg.org/multipage/webappapis.html#update-the-rendering">Update
+    the Rendering</a> step of the Processing Model in the frame, if possible.
```
Comment (chrishtr) [inline index.bs]:
```suggestion
    the Rendering</a> step of the Processing Model in the frame.
```

Comment (chrishtr) [reply index.bs]: Also, s/frame/frame's window's event loop/

File: index.bs [hunk 3]
```diff
@@ -284,6 +284,22 @@ Restrictions and Clarifications {#restrictions}
...
+    <div class=note>
+      As is the case with other elements, contents of an iframe are not
+      painted. With iframes there is an opportunity to skip all of the steps of
+      the Update the Rendering step. The user-agent should skip this work if
```
Comment (chrishtr) [inline index.bs]:
```suggestion
      the Update the Rendering steps. The user-agent should skip this work if
```

File: index.bs (hunk 2 above)
Comment (chrishtr) [reply index.bs]: Re "if possible": I see what you mean now, based on the note below...

File: index.bs (hunk 1 above)
Comment (chrishtr) [reply index.bs]: And all descendant documents in the frame tree

File: index.bs (hunk 2 above)
Comment (vmpstr) [reply index.bs]: I reworded this a bit. frame's window's event loop is extremely confusing to parse... I said frame's event loop, which I think captures sufficient information. WDYT?

Also added "see note below" after "if possible"

**Comment by vmpstr** [comment]
PTAL

**Pull request merged**